integration-test/collection.json

```diff
@@ -154,8 +157,98 @@
 				}
 			},
 			"response": []
+		},
+
+		{
+			"name": "notification",
```

Comment:
This test checks for the notification record existence in database and whether the response from API corresponds, the schema, which is good, but it tests the general function of Strapi. 
I noticed that both Idea-Card and Comment now have lifecycle hooks to create notification when idea/comment are added. I would suggest additional tests to add idea and verify that notification is added successfully, add a comment and verify that notification is added as expected.